src/libs/masq.js

```diff
@@ -1 +1,112 @@
-module.exports = require('masq-client/dist/masq')
+const Masq = require('masq-lib')
+
+const Error = require('../adapters/error').default
+const handleError = (fct, msg, e) => {
+  Error.sendOnce('masq_store', fct, msg, e)
+}
+
+function MasqStore(config) {
+  this.masq = null
+  this.loginLink = null
+  this.config = config
+}
+
+MasqStore.prototype.getAllPois = async function() {
+  const list = await this.masq.list()
+
+  const filteredKeys = Object.keys(list).reduce((filtered, k) => {
+    if (Poi.isPoiCompliantKey(k)) {
+      filtered.push(k)
+    }
+    return filtered
+  }, [])
+
+  if (filteredKeys.length === 0) {
+    return Promise.resolve([])
+  }
+
+  const valuePromises = filteredKeys.map(k => {
+    return this.masq.get(k)
+  })
+
+  const values = await Promise.all(valuePromises).catch(
```

Comment:
i think we can use the try catch syntax here instead of mixin old Promise then & await syntax

Reply:
ok !